h2o-hadoop/h2o-mapreduce-generic/src/main/java/water/hadoop/h2odriver.java

```diff
@@ -1075,6 +1131,43 @@ private void addMapperConf(Configuration conf, String name, String value, byte[]
     mapperConfLength++;
   }
 
+  private static ServerSocket bindCallbackSocket() throws IOException {
+    Exception ex = null;
+    int permissionExceptionCount = 0; // try to detect likely unintended range specifications
+    // eg.: running with non-root user & setting range 100-1100 (the effective range would be 1024-1100 = not what user wanted)
+    ServerSocket result = null;
+    for (int p = driverCallbackPortRange.from; (result == null) && (p <= driverCallbackPortRange.to); p++) {
+      ServerSocket ss = new ServerSocket();
+      ss.setReuseAddress(true);
+      InetSocketAddress sa = new InetSocketAddress(driverCallbackIp, p);
+      try {
+        int backlog = Math.max(50, numNodes * 3); // minimum 50 (bind's default) or numNodes * 3 (safety constant, arbitrary)
```

Comment:
We do not necessarily need to do it. However if we don't specify it the length of the backlog will be automatically set to 50.

Since we know how many clients we have (and we know that first thing they will do is to connect to the driver - possible all at the same time), it seems reasonable to make it a function of number of nodes.

Do you think I should just remove this completely?